symplectic_manifolds.tex

```diff
@@ -38,29 +36,64 @@ \section{Symplectic vector spaces}
+	which is a sort of ``$\Omega$-orthogonal space'' of $W_1$ inside $W$. We can easily prove that $W_1\cap W_1^\Omega=\{0\}$: let $v=ae_1+bf_1\in W_1\cap W_1^\Omega$, then $0=\Omega(v,e_1)=-b$ and $0=\Omega(v,f_1)=a$, therefore $v=0$. 
+	Now we will prove that $W=W_1\oplus W_1^\Omega$. Suppose $v\in W$ has $\Omega(v,e_1)=c$, $\Omega(v,f_1)=d$; then $v=(-cf_1+de_1)+(v+cf_1-de_1)$ where $(-cf_1+de_1)\in W_1$ and $(v+cf_1-de_1)\in W_1^\Omega$. 
+	
+	Let's proceed further. As before, we can select a non-zero $e_2$ from $W_1^\Omega$ for which there must exist a $f_2 \in W_1^\Omega$ such that $\Omega(e_2, f_2) = 1$ up to rescaling, or $e_2$ would fall in $U$ since $U \dir W_1 \dir W_1^\Omega = V$. We understand then our process can go on as long as $W_n \neq W$, giving us a basis which satisfies (2) because each $e_i$ (resp. $f_i$) lives in the ``$\Omega$-orthogonal space'' of the other $e_i$ (resp. $f_i$), and (3) for the same reason plus the fact we make a suitable rescaling at each step.
 \end{proof}
-\begin{corollary}
-\label{cor:can_form}
-	The form $\Omega$, with respect to the basis $B$, is represented as the following block matrix
-	\begin{eqalign}
-		\begin{pmatrix}
-			0 & 0 & 0\\
-			0 & 0 & I_n\\
-			0 & -I_n & 0
-		\end{pmatrix}.
-	\end{eqalign}
-\end{corollary}
 
 \begin{remark}
+	\leavevmode
+	\begin{enumerate}
+		\item The above basis $B$ is not unique.
+		\item
+			The form $\Omega$, with respect to the basis $B$, is represented as the following block matrix
+			\begin{eqalign}\label{eqn:can_form}
+			\begin{pmatrix}
+				0_k & 0 & 0\\
+				0 & 0 & I_n\\
+				0 & -I_n & 0
+			\end{pmatrix}.
+			\end{eqalign}
+			(we omit dimensions of other 0 matrices)
+		\item	The subspace $U$ (and in particular its dimension) does not depend on the choice of the basis. It's an invariant of $(V,\Omega)$.
+		\item	Since $k+2n=m$, $n$ is also a numerical invariant of $(V,\Omega)$.
+	\end{enumerate}
+\end{remark}
+
+\subsection{Symplectic vector spaces}
+
+Let $V$ be an $n$-dimensional $\R$-vector space. Let $\Omega:V\times V\to \R$ be a bilinear map. There is an induced linear map $\tilde\Omega:V\to V^*$ defined by $\tilde\Omega(v)(w):=\Omega(v,w)$. In other words, we can say that $\Omega$ is an element of $V^*\otimes V^*\iso\text{Hom}(V\otimes V,\R)\iso\text{Hom}(V,V^*)$. We have 
+\begin{eqalign}
+\ker\tilde\Omega=\{u\in V\suchthat\Omega(u,v)=0\,\forall\,v\in V\}=U
+\end{eqalign}
+
+\begin{definition}
+	A skew-symmetric bilinear map $\Omega$ is a \textbf{symplectic form} if $\tilde\Omega$ is invertible, i.e. $\ker\tilde\Omega=U=\{0\}$. Then $(V,\Omega)$ is called a \textbf{symplectic vector space}.
```

Comment:
Invertible bilinear maps are called non-degenerate, please revert to my definition.